.github/workflows/build-and-test.yml

```diff
@@ -1,33 +1,66 @@
 name: Build and Test
 
 env:
-  MG_VERSION: '1.4.0'
-  PY_VERSION: '3.8'
+  MG_VERSION: '1.5.0'
   POETRY_VERSION: '1.1.4'
 
 on: [push]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-
+  build_and_test_ubuntu:
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9']
```

Comment:
This array is mentioned twice, can you make it global?

Reply:
I will try it.

Reply:
`The workflow is not valid. .github/workflows/build-and-test.yml (Line: 6, Col: 21): A sequence was not expected .github/workflows/build-and-test.yml (Line: 14, Col: 25): Unrecognized named-value: 'env'. Located at position 1 within expression: env.PYTHON_VERSIONS` env variables cannot be picked up

Reply:
Too bad for that, hope GitHub will add that feature for their Actions :+1: